server/src/services/animal-location.service.ts

```diff
@@ -89,15 +89,22 @@ export class AnimalLocationService {
         });
     }
 
-    getAllAnimalLocationTableData(): any {
+    getAllAnimalsLocationTableData(): any {
 
         const con =  this.databaseService.getConnection();
         return con.then(async (data)=>{
-           //console.log(await data.getRepository(Animal_locations).find())
            return await data.getRepository(AnimalLocation).find(); 
         })  
     }
 
+    getIndividualAnimalLocationTableData(animalID): any {
+
+        const con =  this.databaseService.getConnection();
+        return con.then(async (data)=>{
+           return await data.getRepository(AnimalLocation).find({animalId : animalID}); 
```

Comment:
```suggestion
           return await con.getRepository(AnimalLocation).find({ animalId : animalID }); 
```

Reply:
Done all of the above and added some code to make it work as wanted